.github/CODEOWNERS

```
@@ -126,6 +126,7 @@ tests/appsec/                       @DataDog/asm-python
 tests/contrib/subprocess            @DataDog/asm-python
 tests/snapshots/tests*appsec*.json  @DataDog/asm-python
 tests/contrib/*/test*appsec*.py     @DataDog/asm-python
+tests/contrib/flask/test_flask_pytest_iast.py  @DataDog/asm-python
```

```suggestion
tests/contrib/*/test*iast*.py  @DataDog/asm-python
```

?

to match a bit the one above, maybe same with a snapshot line as well?

---

Commited, thanks for the suggestion.

There's a risk, now, that we own files about ecclesiast, enthusiast or antiasthmatics, but we should manage. 😄